tests/test_link_io_structure.py

```diff
@@ -0,0 +1,121 @@
+from symlink_komodo.sanity_check import read_link_structure, compare_links, check_root_nodes, verify_integrity
+from symlink_komodo.create_links import create_symlinks
+
+import os
+from shutil import copy
+import json
+
+
+def test_read_folder_structure(tmpdir):
+    with tmpdir.as_cwd() as old_dir:
```

Comment:
Would not `with tmpdir.as_cwd()` suffice?

Reply:
:+1: